src/styles/main.css

```
@@ -226,16 +236,15 @@ footer.footer p:not(:last-child) {
 }
 
 .hero-backdrop {
-  position: relative;
-
   width: 100%;
+  height: 100%;
+  background-color: var(--color-bluegray-90);
 
-  background-color: var(--color-bluegray-100);
   background-position: center;
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)